CareWheels/www/app/filelogger/filelogger.html

```diff
@@ -0,0 +1,26 @@
+<ion-view title="File Logger">
+  <ion-content padding="true">
+    <ion-list>
```

Comment:
`<ion-list>` this isn't doing anything right now, if you want to see the styling effects from this you need to have 
```<ion-item>yourListItem</ion-item>```
this inside the `<ion-list>` tags